protonj2-client/src/main/java/org/apache/qpid/protonj2/client/impl/ClientReceiverBuilder.java

```diff
@@ -253,7 +253,7 @@ private ReceiverOptions getDefaultReceiverOptions() {
                     receiverOptions.drainTimeout(sessionOptions.drainTimeout());
                 }
 
-                defaultReceivernOptions = receiverOptions;
+                defaultReceiverOptions = receiverOptions;
```

Comment:
Double-checked locking? https://en.wikipedia.org/wiki/Double-checked_locking#Usage_in_Java

Reply:
Not a concern here as these are immutable and at most this will let one extra copy through before settling